src/main/java/com/faforever/client/replay/ReplayService.java

```diff
@@ -42,4 +42,6 @@
   void enrich(Replay replay, Path path);
 
   CompletableFuture<Integer> getSize(int id);
+
+  void showExternalReplayInfo(Optional<Replay> replay, String replayId);
```

Comment:
Services are not allowed to interact with the user interface. I suggest putting this code into a separate class, like we did for `JoinGameHelper`.